### ISSUE TITLE
Needs support for HACs with districts within them

### ISSUE THREAD

**Issue by blankuserrr** [nitheesh-cpu/HomeAccessCenterAPIv2#6]
![image](https://github.com/user-attachments/assets/668afa0c-aeba-496d-a417-535b73fe57c4)

https://hac.sparcc.org/HomeAccess/Account/LogOn

**Comment by ruskcoder** [comment]
Hey, do you still need this feature?

**Comment by blankuserrr** [comment]
> Hey, do you still need this feature?

I've implemented it myself in a rust rewrite. Unless other users need this, there's no need.

**Comment by ruskcoder** [comment]
Yeah, I was about to say the same, I made my own in nodejs also with session storage capabilities

**Comment by thaddeusm** [comment]
This is definitely crucial in my use case, for what it's worth.

**Comment by blankuserrr** [comment]
I lost my copy but I can recreate it in a couple days

**Comment by ruskcoder** [comment]
> I lost my copy but I can recreate it in a couple days

I have an API at https://api.gradexis.com and you can do for example 

https://api.gradexis.com/hac/classes?username=12345&password=abcde&link=https://hac.org&district=Alliance%20City

It is open source so you can see my account and use it if you want
